### ISSUE TITLE
Train with PRE_ORI : 'False'

### ISSUE THREAD

**Issue by mahiratmis** [Royalvice/DocDiff#32]
when I train the network with PRE_ORI : 'True' everything looks fine. I tried training the model with PRE_ORI : 'False'. However, during trainig where I save the initial predictions during epochs I observe that initial predictions have some kind of white layer on them. Do you have an idea of why it could be so ?

You can see the attached image for better illustration of the problem. The results are from a batch of images. Left picture is ground truth, middle picture is initial prediction and right picture is (ground truth - inital prediction).

![Ekran görüntüsü 2024-06-09 224106](https://github.com/Royalvice/DocDiff/assets/1335306/ff154fad-9da3-41bd-b650-4d5450b36f0e)

**Comment by Royalvice** [comment]
You're right. When PRE_ORI: 'False', the visualization code during training is incorrect.

**Comment by mahiratmis** [comment]
Would it affect the inference ? If inference uses the results of the initial prediction the inference will also yield in bad results.

**Comment by mahiratmis** [comment]
The inference also produces results with white layer. Can you help adress the problem ?
